proto/cosmos/evidence/v1beta1/query.proto

```diff
@@ -4,22 +4,27 @@ package cosmos.evidence.v1beta1;
 import "cosmos/base/query/v1beta1/pagination.proto";
 import "gogoproto/gogo.proto";
 import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
 
 option go_package = "github.com/cosmos/cosmos-sdk/x/evidence/types";
 
 // Query defines the gRPC querier service.
 service Query {
   // Evidence queries evidence based on evidence hash.
-  rpc Evidence(QueryEvidenceRequest) returns (QueryEvidenceResponse) {}
+  rpc Evidence(QueryEvidenceRequest) returns (QueryEvidenceResponse) {
+    option (google.api.http).get = "/cosmos/evidence/v1beta1/evidences/{evidence_hash}";
```

Comment:
```suggestion
    option (google.api.http).get = "/cosmos/evidence/v1beta1/evidence/{evidence_hash}";
```

I know it should be plural, but `evidence` can actually be plural too, see https://github.com/cosmos/cosmos-sdk/pull/6978#discussion_r467893440. Let's keep it consistent with other places.